libs/portals/admin/petition/src/components/ListActions/UpdateList/UpdateList.action.ts

```diff
@@ -0,0 +1,54 @@
+import { WrappedActionFn } from '@island.is/portals/core'
+import {
+  UpdateListDocument,
+  UpdateListMutation,
+  UpdateListMutationVariables,
+} from '../../../shared/mutations/updateList.generated'
+import parseDate from 'date-fns/parse'
+
+export const transformDate = (val: string) =>
+  parseDate(val, 'dd.MM.yyyy', new Date())
+
+export const updateListAction: WrappedActionFn = ({ client }) => async ({
+  request,
+  params,
+}) => {
+  const formData = await request.formData()
+
+  const openedDate = formData.get('openedDate') as string
+  const closedDate = formData.get('closedDate') as string
+
+  const description = formData.get('description') as string
+  const title = formData.get('title') as string
+
+  if (!params['listId']) {
+    throw new Error('something went wrong')
```

Comment:
Is this error shown to the user?
If not, can it be more specific for the sake of logs?
`Listid not provided or found in params`?